### ISSUE TITLE
Connections to Hyper-V virtual machines?

### ISSUE THREAD

**Issue by jol64** [Terminals-Origin/Terminals#190]
maybe I missed it, but I haven´t discovered an option to connect to virtual machines. Is that doable? Or can that be added?
Thanks, Joachim

**Comment by jirkapok** [comment]
Yes, there is no such support. Since it was already required, any implementation is welcome.
